src/VolleyManagement.Backend/VolleyManagement.UI/App_Start/BundleConfig.cs

```diff
@@ -1,7 +1,8 @@
 namespace VolleyManagement.UI
 {
     using System.Web.Optimization;
-
+    
+#pragma warning disable S1075 // URIs should not be hardcoded
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)